src/drive/web/modules/views/OnlyOffice/useConfig.js

```diff
@@ -32,6 +33,13 @@ const useConfig = () => {
     [fetchStatus]
   )
 
+  useEffect(
+    () => {
+      setConfig()
+    },
+    [isEditorForcedReadOnly]
+  )
```

Comment:
Why this useEffect? Since `isEditorForcedReadOnly` is present in the dependencies array https://github.com/cozy/cozy-drive/pull/2365/commits/fe2b9b17482b5acd8020d1518b7ca45fab52914a#diff-b1ff6a384c21483129ebd8acd7256feb98863a5d107463589d6688034352afa3R106, the next useEffect will be called, right? 

Same question for the previous useEffect (the one with the fetchStatus)

Reply:
C'est pour isoler le fait que si `isEditorForcedReadOnly` change, on reset la config.

C'est pour isoler les useEffect selon ce qu'ils font, pour ne pas avoir un useEffect qui set trop de choses en même temps, ou avec trop de condition.

J'avais lu qu'il valait mieux faire comme ça pour plus de clarté quand on utilise les useEffect (les contextualiser). Je n'ai plus l'article sous la main mais je peux chercher un peu si nécessaire.